tests/PhpPact/Standalone/StubServer/StubServerTest.php

```diff
@@ -15,15 +15,13 @@ public function testStartAndStop()
     {
         try {
             $pactLocation = __DIR__ . '/../../../_resources/someconsumer-someprovider.json';
-            $host         = 'localhost';
             $port         = 7201;
             $endpoint     = 'test';
 
             $subject = (new StubServerConfig())
-                ->setPactLocation($pactLocation)
-                ->setHost($host)
+                ->setFiles($pactLocation)
                 ->setPort($port)
-            ->setEndpoint($endpoint);
+                ->setEndpoint($endpoint);
```

Comment:
`$endpoint` is not useful if we interact with `StubServer` directly like this. It's only useful when we interact with stub service via `StubServerHttpService`.

I think it's better if we move `$endpoint` to `StubServerHttpService::getJson` like this:

```php
// class StubServerHttpService
public function getJson(string $endpoint): string
    {
        $uri      = $this->config->getBaseUri()->withPath('/' . $endpoint);
        $response = $this->client->get($uri, [
            'headers' => [
                'Content-Type' => 'application/json',
            ],
        ]);

        return \json_encode(\json_decode($response->getBody()->getContents()));
    }
```

Do you think so?

Reply:
See https://github.com/pact-foundation/pact-php/pull/296